docs/_getting-started/index.md

```diff
@@ -277,30 +436,36 @@ Follow the same step as the IPC Check account balance
 
 A successful call will return a response below:
 
-![Create new account command](../../static/images/rpclistaccounts.png)
+```
+{"jsonrpc":"2.0","id":5,"result":["0xca57f3b40b42fcce3c37b8d18adbca5260ca72ec"]}
+```
 
 
 **→ Send ETH to accounts**
 
 **Syntax:**
-```shell
+```javscript
     curl -X POST http://http://127.0.0.1:8545 \
         -H "Content-Type: application/json" \
        --data '{"jsonrpc":"2.0", "method":"eth_sendTransaction", "params":[{"from": "<ADDRESS_1>","to": "<ADDRESS_2>","value": "0x9184e72a"}], "id":1}'
 ```
 
 You need to convert eth to wei and get the hex value to send a transaction.
-For example:  0.0241 ether is 24100000000000000 wei, and would be encoded as the hex string "0x559ed283164000" in the JSON-RPC API.
+
+> **Example:**  0.0241 ether is 24100000000000000 wei, and would be encoded as the hex string "0x559ed283164000" in the JSON-RPC API.
 
 **step 3:** Run the command below
 
-```shell
+```javscript
```

Comment:
You have added the `javscript` language marker to all code snippets, but it's not appropriate:

- First, it's `javascript` not `javscript`
- Second, `shell` is fine for command-line examples.